components/gitpod-db/src/typeorm/migration/1592203031938-Baseline.ts

```diff
@@ -81,7 +81,7 @@ export class Baseline1592203031938 implements MigrationInterface {
         {
             const exists = (await queryRunner.query(`SELECT COUNT(1) AS cnt FROM d_b_user WHERE id = 'builtin-user-workspace-probe-0000000'`))[0].cnt == 1;
             if (!exists) {
-                await queryRunner.query(`INSERT IGNORE INTO d_b_user (id, creationDate, avatarUrl, name, fullName) VALUES ('builtin-user-workspace-probe-0000000', '${new Date().toISOString()}', '', '${BUILTIN_WORKSPACE_PROBE_USER_NAME}', '')`)
```

Comment:
Is this guaranteed, or do we have environments where there builtin user has another ID (for whatever reason)?
I know it seems obvious from the code. I just assume there was some motivation to use the user name in the first place, and want to be sure we don't miss sth here. :slightly_smiling_face:

Reply:
@csweichel

Reply:
We have both in staging and prod one user with that id and multiple users (3 in staging, 4 in prod) with that name.